### DISCUSSION TITLE
[Bug]Pinned repo scroll view

### DISCUSSION THREAD

**Discussion by Hephaest** [community/community#2842] [Mobile]
<p align="center">
<img width="600" src="https://user-images.githubusercontent.com/37981444/101300185-96181380-37e9-11eb-986b-f469631ad0de.gif">
</p>

**[GitHub Mobile][iOS]**
This issue happened yesterday. It is unacceptable and I think GitHub iOS developer should check out this issue.

**Additional Info**
GitHub for iOS, Version 1.3.3 (21290296)
Device: iPhone Xs Max (iOS 14.2)

**Comment by candyho** [comment]
Hey, we are aware of this bug and have an issue to track this bug in our internal repo. Will keep you posted when this is fixed. Related thread for FYR - https://github.com/github/feedback/discussions/2805

**Reply by Hephaest** [reply]
Okay, but I want to highlight that this .gif file reveals 2 problems to make it clear:
1. Wrong offset for scroll view. (Same with #2805)
2. Duplicate pinned repo cards.
